<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Querys</string>
  </property>
  <widget class="QPlainTextEdit" name="QueryBrowseAndEdit_PTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_btn">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>233</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>ESC키 : 자동저장 및 종료</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
